src/models/comment.js

```diff
@@ -0,0 +1,22 @@
+//Requires Mongoose
+const mongoose = require("mongoose");
+
+//Creates variables to store Schema
+
+const Schema = mongoose.Schema;
+
+//Creates comment schema
+
+const commentSchema = Schema({
+  userID: { type: Schema.Types.ObjectId, ref: "User" },
+  username: { type: Schema.Types.ObjectId, ref: "User" },
```

Comment:
Do we need both userId and username?  We just need to reference it once and then we can return all the necessary things in the